webpack.config.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+/* eslint-env node */
+
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  devtool: 'source-map',
+
+  entry: [
+    path.resolve(__dirname, 'src', 'sidebar', 'app', 'app.js'),
+  ],
+
+  output: {
+    // build to the extension src vendor directory
+    path: path.resolve(__dirname, 'build'),
+    filename: path.join('sidebar', 'app.js'),
+  },
+
+  plugins: [
+    // cleanup build folder
+    new CleanWebpackPlugin([
+      'build'
+    ]),
+    // Moves files
+    new CopyWebpackPlugin([
+      { from: path.join('src') }
+    ], {
+      ignore: [
+          path.join('sidebar', 'app', '**', '*'),
+          path.join('sidebar', 'static', 'scss', '**', '*')
+      ],
+    }),
+  ],
+
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: ['babel-loader?presets[]=env&presets[]=react&presets[]=stage-2&sourceMaps=true'],
```

Comment:
Babel compile jsx syntax to js. Sound confusing since we store jsx code in js files (could be .jsx to be more explicit but react use .js by default). I will add a comment 👍.